projects/plugins/jetpack/class-jetpack-stats-dashboard-widget.php

```diff
@@ -51,7 +51,9 @@ public static function wp_dashboard_setup() {
 		 *
 		 * @param bool Whether to show the widget to the current user.
 		 */
-		if ( ! apply_filters( 'jetpack_stats_dashboard_widget_show_to_user', current_user_can( 'view_stats' ) ) ) {
```

Comment:
We need to keep the filter as we talked 😄

Reply:
Updated in [166745a](https://github.com/Automattic/jetpack/pull/41549/commits/166745a2b4f191151c57c07d6ab2c14002cc5868).